### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Abeer-Abdelhamid**
Add My Card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -107,6 +107,36 @@ <h2>A project for first-time contributions</h2>
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:110]: Replace `TEMPLATE: MAKE A COPY` with your name

File: index.html [hunk 2]
```diff
@@ -107,6 +107,36 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://www.w3schools.com/java/java_oop.asp" target="_blank" title="OOP">
+                  What is OOP?
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:139]: Replace `END TEMPLATE` with your name

**Review by TenTraicion — CHANGES_REQUESTED** [review]
make sure to run `git pull before making any changes, and solve the issues before committing /pushing to the repo. Thanks!

File: index.html (hunk 1 above)
Comment (Abeer-Abdelhamid) [reply index.html]: It's already done, thanks.

**Pull request merged**